mkosi.sanitizers/mkosi.extra/usr/lib/systemd/system.conf.d/10-sanitizers.conf

```diff
@@ -1,6 +1,7 @@
 # SPDX-License-Identifier: LGPL-2.1-or-later
 
 [Manager]
-DefaultEnvironment=ASAN_OPTIONS=strict_string_checks=1:detect_stack_use_after_return=1:check_initialization_order=1:strict_init_order=1:disable_coredump=0:use_madv_dontdump=1 \
-                   UBSAN_OPTIONS=print_stacktrace=1:print_summary=1:halt_on_error=1 \
-                   LSAN_OPTIONS=suppressions=/usr/lib/systemd/leak-sanitizer-suppressions
+DefaultEnvironment= \
+        ASAN_OPTIONS=strict_string_checks=1:detect_stack_use_after_return=1:check_initialization_order=1:strict_init_order=1:detect_invalid_pointer_pairs=2:handle_ioctl=1:print_cmdline=1:disable_coredump=0:use_madv_dontdump=1:log_to_syslog=1 \
```

Comment:
Why add log_to_syslog=1? The output will end up in the journal either way no?

Reply:
I added it because we did. Maybe for early boot stage? Not sure.
https://github.com/systemd/systemd/blob/560412148fa62d779ea522d43960dd3acd44b261/test/test-functions#L924-L928

Reply:
This is the default environment though, not the manager environment.

Reply:
Ugh, indeed.

Reply:
Tentatively dropped. I will re-add it at the correct place later if it is necessary for PID1.